R/logger.R

```diff
@@ -0,0 +1,797 @@
+##
```

Comment:
this file also has some extra white line but I did not address them as this function is directly coming from package function.